llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -1623,6 +1623,8 @@ bool RISCVFrameLowering::assignCalleeSavedSpillSlots(
     if ((unsigned)FrameIdx > MaxCSFrameIndex)
       MaxCSFrameIndex = FrameIdx;
     CS.setFrameIdx(FrameIdx);
+    if (RISCVRegisterInfo::isRVVRegClass(RC))
+      MFI.setStackID(FrameIdx, TargetStackID::ScalableVector);
```

Comment:
The spill size is still an unsigned, and not a TypeSize. How do scalable types still have fixed spill sizes?

Reply:
Good question. Do you want me to investigate this issue and may be create a fix in current PR?

Reply:
Definitely should not be fixed in this PR. It requires tablegen changes, TargetRegisterInfo.h interface changes. Probably individual changes other targets.